EIPS/eip-5005.md

```diff
@@ -0,0 +1,225 @@
+---
+eip: 5005
+title: Zodiac
```

Comment:
```suggestion
title: Zodiac Avatar Accounts and Extensions
```

I feel like the core contribution of this EIP is Avatar accounts, and I think "extensions" covers modifiers, modules, and guards well enough?

Reply:
I'd prefer not to introduce additional terminology that isn't used elsewhere.

If we were to add anything to the title, I'd be inclined to go with something like "Zodiac Avatar Accounts and Mods".

Thoughts?

Reply:
Hm, how about just `Zodiac Avatar Accounts`?

Reply:
That seems reasonable as well.
One thing I have been conscious of lately is that this pattern is useful beyond just DAOs. So I wonder if it would be worthwhile changing the rest of the language to be less opinionated about how it is used / what it is used for?

Reply:
Added this PR to generalize the language. This feels like the right direction to me.
https://github.com/auryn-macmillan/EIP-Zodiac/pull/1